exercises/K-functions-parameters/exercise2.js

```diff
@@ -1,4 +1,6 @@
-// Declare your function first
+function divide(num1, num2){
+    return num1/num2;
```

Comment:
You could name these variables "numerator" and "denominator" so as not to get them confused

Reply:
Done